CHANGELOG.rst

```diff
@@ -24,6 +24,7 @@ Removed
 Fixed
 -----
 - Fixed exporting NLU training data in ``json`` format from ``rasa interactive``
+- Fixed numpy deprecation warnings showing in ``rasa`` installed with ``pip``
```

Comment:
how else would you install rasa than with pip?
```suggestion
- Fixed numpy deprecation warnings
```

Reply:
from source? 😉 was trying to make it clear why it showed up installed with pip but we didn't run into it when developing locally

but i guess since it's mostly a public thing we can remove that